M2/Macaulay2/packages/Text.m2

```diff
@@ -816,6 +816,16 @@ document { Key => {(validate, Hypertext),validate},
 	  eliminated, or a more-final type of hypertext, convertible immediately to HTML, will have to be developed."
 	  }
      }
+document { Key => {(style, Hypertext),style},
+    Usage => "style (x,opts)",
+    Inputs => { "x" => { TO "hypertext" } },
+    Consequences => { { "Returns a new Hypertext object with its style option updated." }},
+    EXAMPLE lines ///
+    d = DIV { "Hello" }
+    d = style(d, "font-weight" => "bold")
+    peek d
+    ///
+    }
```

Comment:
I guess the code for `style` could also be in the Text package.

Reply:
Oops! Will fix that

Reply:
edited: oh. I thought the comment was about the documentation. let me think about it carefully before I make this change.

Reply:
do you have an opinion Mahrud? since you're the one who restructured `hypertext.m2`.
In principle it would make sense to move `style` to `Text`, but it's a bit weird since at the moment `Text` is a completely empty package which only contains exports & documentation.

Reply:
I do not.

Reply:
OK then I'd rather not move style.